### PR TITLE
Added a multiple dice implementation to UI

### PR DESCRIPTION
**Pull request by FranciscoGaraventa**
I add two new widget, one contains the implementation of the posible dice events and the second implements the roll action.
An example with 16 dices:
16Dice
An example with 38 dices:
ScrollViewDices

Result of Unit Test:
dice_bloc_test
dice_screen_test

### CODE REVIEW

File: lib/src/app.dart [hunk 1]
```diff
@@ -10,13 +10,13 @@ class MyApp extends StatefulWidget {
...
 class _MyAppState extends State<MyApp> {
-  final _blocs = <Bloc>[];
+  final _blocs = <DiceBloc>[];
 
   @override
   void initState() {
     super.initState();
-    DiceBloc bloc = DiceBloc();
-    _blocs.add(bloc);
+    _blocs.add(new DiceBloc());
```
Comment (luisjavierglb) [inline lib/src/app.dart]: We shouyld be creating more than one instance of a Bloc, the logic for both dices should be handled inside the DiceBloc

File: lib/src/screen/dice/dice_screen.dart [hunk 2]
```diff
@@ -30,35 +28,19 @@ class _DiceScreenState extends State<DiceScreen> {
...
-        ),
-        floatingActionButton: OnPressWidget(
-          diceBloc: widget.diceBloc,
+            child: ListView.builder(
+              shrinkWrap: true,
+              scrollDirection: Axis.horizontal,
+              itemCount: widget.diceBloc.length,
+              itemBuilder: (BuildContext context, int index) {
+                return DiceStreamBuilder(
+                  diceBloc: widget.diceBloc[index],
```
Comment (luisjavierglb) [inline lib/src/screen/dice/dice_screen.dart]: As I mentioned before, this is not a good practice, we should have the logic for all the dices inside just one Bloc

File: lib/src/screen/dice/dice_screen.dart [hunk 3]
```diff
@@ -30,35 +28,19 @@ class _DiceScreenState extends State<DiceScreen> {
...
-                    ),
-                  ),
-                ],
-              );
-            },
-          ),
-        ),
-        floatingActionButton: OnPressWidget(
-          diceBloc: widget.diceBloc,
+            child: ListView.builder(
```
Comment (luisjavierglb) [inline lib/src/screen/dice/dice_screen.dart]: Using a listview, is probably overkill for this particular scenario

**Review by luisjavierglb — CHANGES_REQUESTED** [review]

File: lib/src/app.dart (hunk 1 above)
Comment (FranciscoGaraventa) [reply lib/src/app.dart]: Done.

File: lib/src/screen/dice/dice_screen.dart (hunk 2 above)
Comment (FranciscoGaraventa) [reply lib/src/screen/dice/dice_screen.dart]: Done.

File: lib/src/screen/dice/dice_screen.dart (hunk 3 above)
Comment (FranciscoGaraventa) [reply lib/src/screen/dice/dice_screen.dart]: I change the ListView with a Wrap and a SingleChildScrollView

**Review by MarcosCGlobant — COMMENTED** [review]
You should rename the PR, and the commit title, because your implementation is wider than just 2 dice

File: Test/dice_bloc_unit_test.dart [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import 'package:first_flutter_app/src/RollEvent.dart';
+import 'package:first_flutter_app/src/bloc/dice_bloc.dart';
```
Comment (MarcosCGlobant) [inline Test/dice_bloc_unit_test.dart:2]: Use relative imports with your project files.

File: Test/dice_screen_widget.dart [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import 'package:first_flutter_app/src/app.dart';
+import 'package:first_flutter_app/src/widgets/roll_events.dart';
```
Comment (MarcosCGlobant) [inline Test/dice_screen_widget.dart:2]: Use relative imports with your project files.

**Review by MarcosCGlobant — APPROVED** [review]
LGTM, nice job

**Review by luisjavierglb — APPROVED** [review]
LGTM

**Pull request merged**